docs/monitoring/tag-support.md

```diff
@@ -1,125 +1,113 @@
+Below tags are sent as part of Network Traffic Status metrics:
 
-Below are the steps to configure custom tags in Sematext Agents.
+| Tag Name  | Description  | Related reserved tags
+|:--|:--|:--
+| network.topology.address | Local address of the network connection | network.connections.address 
+| network.topology.destination.address | Remote host address of the network connection | network.connections.dst.address
+| network.topology.destination.port | Remote port |
+| network.topology.protocol |  Protocol name (TCP or UDP) |
+| network.topology.outgoing | determines whether the connection is incoming (client connected to server) or outgoin(current machine connects to external server) |
+| network.topology.process.id | process's identifier that produces connection stats |
+| network.topology.process.name | name of the process's image that produces connection stats | 
 
-#### Sematext App Agent
+### Serverless Tags
 
-To add custom tags for each app edit the below property in the monitor configuration file: 
-`/opt/spm/spm-monitor/conf/spm-monitor-config-${token}-${jvm}.properties`:
+Below are Serverless related tags sent as part of metrics/logs in the Serverless environment.
 
-``` properties
-# add tags if you want to use them, example: SPM_MONITOR_TAGS="env:foo, role:bar"
-SPM_MONITOR_TAGS="env:dev, project:projectName, role:webfrontend"
-```
+| Tag Name  | Description  | Related reserved tags
+|:--|:--|:--
+| function.name |  | 
 
-The name and value of custom tags should match this regex: `[a-zA-Z0-9_\-=\+\.]*`.
 
-#### Sematext Docker Agent
+### Other Special Tags
 
-Tags are provided in the environment variable SPM\_MONITOR\_TAGS for example:
+Below are the tags that are reserved for future use.
 
-``` bash
-docker run -e SPM_MONITOR_TAGS="env:dev, project:projectName, role:webfrontend" ... sematext/sematext-agent-docker
-```
+| Tag Name  | Description  | Related reserved tags
+|:--|:--|:--
+| source |  Represents the source of the event  | facility
+| service | Service that generated the event | service.id, service.name
+| span | Building block of trace in distributed tracing | 
+| error.code | Error code | error.id, error.message
 
-#### Sematext Node Agent
 
-Tags could be configured in the config file `./.spmagentrc` or
-`/etc/spmagentrc`
+## Defining Tags in Sematext App Agent Integration YAMLs
 
-``` properties
-SPM_MONITOR_TAGS = env:dev, project:projectName, role:webfrontend
-```
+In App Agent, tags are extracted automatically from metric data sources & values based on YAML configuration. The tags can be configured in `tag` section App Agent integration YAMLs.  The maximum allowed length for the tag name is 200 characters. The tag name should match this regex: <nobr>`[a-zA-Z0-9_\-.:(\\ |,=)]+`</nobr>. Examples of these tags are hostname, port, webapp name, jvm name, disk, elasticsearch index, etc. You don't need to adjust these tags for built-in
```

Comment:
In App Agent, tags are extracted automatically from metric data sources & values based on YAML configuration. The tags can be configured in `tag` section App Agent integration YAMLs.
=>
In the App Agent, tags are extracted automatically from metric data sources & values based on the YAML configuration. Tags can be configured in the `tag` section of the App Agent integration YAMLs.

Reply:
done